_locales/en/messages.json

```diff
@@ -66,7 +66,7 @@
     "message": "For website"
   },
   "confirmPreset": {
-    "message": "Are you sure you want to load this preset?"
+    "message": "Load this preset and overwrite current settings?"
```

Comment:
We could possibly change it to `current addon settings` to clarify that settings from other addon settings can't be affected by setting a preset. What do you think?

Reply:
Done.

Reply:
@RedGuy12 Please confirm if you still approve this